roxctl/image/scan/cve.go

```diff
@@ -119,6 +122,9 @@ func getVulnerabilityJSON(vulnerabilities []*storage.EmbeddedVulnerability, comp
 			CveID:                 vulnerability.GetCve(),
 			CveSeverity:           severity,
 			CveInfo:               vulnerability.GetLink(),
+			CveCVSS:               vulnerability.GetCvss(),
```

Comment:
biggest nit ever:  Everywhere else you put CVSS above its link.  For code consistency you could do that here as well.

Reply:
I thrive on nits, so I'm good with this